src/main/java/io/github/linead/nametags/Nametags.java

```diff
@@ -66,18 +77,40 @@ public static void main(String[] args) throws Exception {
                 continue;
             }
 
-
             //new attendee
             Attendee att = new Attendee();
             att.setId(rsvp.getMember().getMember_id());
             att.setName(rsvp.getMember().getName());
+
             //get member
             Members.Member member = members.get(rsvp.getMember().getMember_id());
             if(member != null) {
                 att.setJoined(member.getJoined());
                 if(member.getPhoto() != null) {
-                    att.setPictureUrl(member.getPhoto().getThumb_link());
+
+                    String pictureURLStr = member.getPhoto().getThumb_link();
+                    log.info("getAttendeeList() photo URL:" + pictureURLStr);
+
+                    if(sendPictureAsBase64) {
+                        // Retrieve image data and encode it
+                        try {
+                            byte[] b = IOUtils.toByteArray((new URL(pictureURLStr)).openStream());
+                            Base64 b64 = new Base64();
```

Comment:
PS. Saved a dependency too as I was using the Apache Common Codec implementation. Now using built-in Java 8 encoder.